lib/validators/liveValidator.ts

```diff
@@ -36,7 +36,7 @@ export interface LiveValidatorOptions {
   }
   useRelativeSourceLocationUrl?: boolean
   directory: string
-  swaggerPathsPattern: string
+  swaggerPathsPattern: string[]
```

Comment:
Can this be string | string[]?

Reply:
It can but it's not allowed to pass by chart in validation service.